examples/example-blog/components/meta.tsx

```diff
@@ -14,6 +14,7 @@ export function Meta({ title, tags }: MetaProps) {
   return (
     <Head>
       <link
+        key="head_link_canonical"
```

Comment:
Let's not prefix our key's with `head_` or even with `link_` or `meta_`. It seems overly verbose. If React is trying to dedupe a `link` element, it doesn't need the extra specificity of a prefix that duplicates the HTML structure.

tldr; let's use `key="canonical"` here.